packages/hash/shared/src/entity.test.ts

```diff
@@ -27,38 +27,38 @@ describe("flatMapTree", () => {
     }
 
     expect(result).toMatchInlineSnapshot(`
-Array [
-  Object {
-    "children": Object {
-      "relatesTo": Object {
-        "entityId": "id2",
-      },
-    },
-    "rootId": "1",
-  },
-  "1",
-  Object {
-    "relatesTo": Object {
-      "entityId": "id2",
-    },
-  },
-  Object {
-    "entityId": "id2",
-  },
-  "id2",
-  Object {
-    "children": Object {
-      "entityId": "xid1",
-    },
-    "rootId": "2",
-  },
-  "2",
-  Object {
-    "entityId": "xid1",
-  },
-  "xid1",
-]
-`);
+      [
```

Comment:
Related to [Jest 29: Snapshot format changes](https://jestjs.io/blog/2022/08/25/jest-29)